File: Java/Zliczanie liczb i wyrazow.java
import java.util.Scanner;

class Main
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        while (input.hasNextLine()) 
        {
            String[] data = input.nextLine().split(" ");
            int numbers = 0;
            int words = 0;
            for (String word : data) 
            {
                boolean isNumber = true;
                for (char symbol : word.toCharArray()) 
                {
                    if (!Character.isDigit(symbol)) 
                    {
                        isNumber = false;
                        break;
                    }
                }
                if (isNumber)
                    numbers++;
                else
                    words++;
            }
            System.out.println(numbers + " " + words);
        }
        input.close();
    }
}

File: Python/Zliczanie liczb i wyrazow.py
while True:
    numbers = 0
    words = 0
    try:
        sentence = input().split()
        for word in sentence:
            try:
                int(word)
                numbers += 1
            except ValueError:
                words += 1
        print(f"{numbers} {words}")
    except:
        break
